IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -252,19 +252,18 @@ - (MSIDIdToken *)getIDTokenForAccount:(MSIDAccountIdentifier *)account
                                                  error:error];
 }
 
-- (NSArray<MSIDAccount *> *)allAccountsForEnvironment:(NSString *)environment
-                                             clientId:(NSString *)clientId
-                                             familyId:(NSString *)familyId
-                                              context:(id<MSIDRequestContext>)context
-                                                error:(NSError **)error
+- (NSArray<MSIDAccount *> *)allAccountsForConfiguration:(MSIDConfiguration *)configuration
+                                               familyId:(NSString *)familyId
+                                                context:(id<MSIDRequestContext>)context
+                                                  error:(NSError **)error
```

Comment:
nit: we don't use all fields of the configuration, would it be clearer if we have parameters be more specific?
i.e./ authority & clientId instead of the configuration?

Reply:
I agree with JK here, I think it is better to send specific parameters.

Reply:
fixed